src/CounterLabel.js

```diff
@@ -1,25 +1,49 @@
-import React from 'react'
 import PropTypes from 'prop-types'
-import classnames from 'classnames'
-import sass from 'sass.macro'
-import {injectGlobal} from 'emotion'
 import styled from 'styled-components'
-import {COMMON} from './constants'
+import {COMMON, Base, get} from './constants'
 import theme from './theme'
 
-injectGlobal(sass`
-  @import "primer-support/index.scss";
-  @import "primer-labels/lib/counters.scss";
-`)
+const colorStyles = ({scheme, ...props}) => {
+  return {
+    color:
+      scheme === 'gray-light'
+        ? get('colors.gray.9')(props)
```

Comment:
Notice in functions like these, you have to explicitly call props on the get function